orangecontrib/prototypes/widgets/owsplit.py

```diff
@@ -0,0 +1,136 @@
+import Orange.data
+import numpy as np
+from Orange.data import Table
+from Orange.data.util import SharedComputeValue, get_unique_names
+from Orange.widgets import gui
+from Orange.widgets.settings import ContextSetting, DomainContextHandler
+from Orange.widgets.widget import OWWidget, Msg, Output, Input
+from AnyQt.QtCore import Qt
+
+from Orange.data import DiscreteVariable, StringVariable
+from Orange.widgets.utils.itemmodels import DomainModel
+from Orange.widgets.utils.widgetpreview import WidgetPreview
+from orangewidget.settings import Setting
+
+
+def get_string_values(data, var):
```

Comment:
This is totally a question of style but: consider changing this function into a `@staticmethod` of class `SplitColumn`. It is use only there, so there's no need to pollute the module's namespace with it.